modules/Microsoft.ApiManagement/service/readme.md

```diff
@@ -266,6 +267,14 @@ userAssignedIdentities: {
 
 - *None*
 
+## Cross-referenced modules
+
+This section gives you an overview of all local-referenced module files (i.e., other CARML modules that are referenced in this module) and all remote-referenced files (i.e., Bicep modules that are referenced from a Bicep Registry or Template Specs).
+
+| Reference | Type |
+| :-- | :-- |
+| `Microsoft.ApiManagement/authorizationServers` | Local reference |
```

Comment:
There's another comment about this. Do we need to explicitily call out 'remote reference'? What impact does it have on the user as shouldn't the API version table already contain the entire information of other types?

Reply:
As far as I'm concerened there are 2 good reasons:
- Information on external dependencies is always good. If you reference a public endpoint it's 'easy' but what if it is a non-punlic private registry/template spec? Would you not want to be aware as a consumer?
- What if you're in an environment without public internet access (not the most uncommon scenario)?

Reply:
Thanks for clarifying. I know we do not have those use cases in our CI, at least today. And if we are going to support this feature, we need good documentation to explain the different reference types with examples. I couldn't see it in our doco for cross reference.

Reply:
We don't use them in our CI, that's correct - but we never not supported them. 
Consumers however may. This feature borderline only surfaces these references if you choose to leverage them. I don't think we have any plans to do it ourselves though.

Reply:
As long as we have it as a feature. We should surface some wording about what does local reference vs remote reference mean and when would the consumer see them and why. Would be useful if you can add that please.

Reply:
Isn't the dislaimer exactly that? 
> This section gives you an overview of all local-referenced module files (i.e., other CARML modules that are referenced in this module) and all remote-referenced files (i.e., Bicep modules that are referenced from a Bicep Registry or Template Specs).

Reply:
Ah ok sorry I didn't easily spot this as it's only visible when there's a cross reference. I actually meant in the cross reference wiki for someone reading that documentation.

Reply:
Adjusted the description in the docs slightly to align better with the readme